packages/editor-ui/src/App.vue

```diff
@@ -160,6 +161,9 @@ export default mixins(
 
 			this.trackPage();
 		},
+		'$store.getters.defaultLocale'(newLocale) {
```

Comment:
did this not work?
```suggestion
		defaultLocale(newLocale) {
```

Reply:
The watcher cannot read from the mapped getter directly:

<img width="1433" alt="image" src="https://user-images.githubusercontent.com/44588767/161282337-8ec95306-2efb-43e1-a989-7e6d5f39a1d1.png">


Either using the full store path as I did, or adding an extra getter just for the watcher.

```js
defaultLocale(): string {
	return this.$store.getters.defaultLocale;
},
```